.travis.yml

```diff
@@ -14,30 +14,38 @@ env:
 before_install:
   # Install .NET Core SDK.
   - if [[ $LANG == *"c_sharp"* ]] || [[ $LANG == *"visual_basic"* ]]; then
+      set -e;
```

Comment:
Shall we perhaps wrap only the beginning and the end of the `before_install` wrap?